src/views/ConsentContinuationOut.vue

```diff
@@ -231,6 +283,15 @@ export default class ConsentContinuationOut extends Vue {
   private isCertified = false
   private certifyFormValid: boolean = null
 
+  // variables for Courder Order POA component
+  private fileNumber = ''
+  private hasPlanOfArrangement: boolean = null
+  private courtOrderValid: boolean = null
+
+  // variables for Document Delivery component
+  private documentDeliveryValid: boolean = true
```

Comment:
TS can infer that this is a boolean because you assign an initial value of true. So please remove the typing here (sometimes lint complains about redundant typing).